src/Registry.ts

```diff
@@ -201,12 +213,12 @@ export class Registry extends Evented<{}, RegistryLabel, RegistryEventObject> im
 		return null;
 	}
 
-	public getInjector<T extends Injector>(label: RegistryLabel): T | null {
+	public getInjector<T>(label: RegistryLabel): InjectorItem<T> | null {
 		if (!this._injectorRegistry || !this.hasInjector(label)) {
 			return null;
 		}
 
-		return this._injectorRegistry.get(label) as T;
+		return this._injectorRegistry.get(label)!;
```

Comment:
In the return were `InjectorItem<T> | undefined` wouldn't we be able to avoid the `!`?

Reply:
We already check whether the item exists, so I know at this point it is not undefined and I wanted to avoid changing the API any more than required.

Reply:
@devpaul ^